pkg/ioutils/writers_test.go

```diff
@@ -6,6 +6,32 @@ import (
 	"testing"
 )
 
+func TestWriteCloserWrapperClose(t *testing.T) {
+	called := false
+	writer := bytes.NewBuffer([]byte{})
+	wrapper := NewWriteCloserWrapper(writer, func() error {
+		called = true
+		return nil
+	})
+	err := wrapper.Close()
+	if err != nil {
```

Comment:
@runcom yeah but these test are not (yet) using gocheck, so can't do for now.. But i'll definitely look at how to use gocheck for these tests too (not for the PR :)).